design-toolkit/components/src/components/tree/index.tsx

```diff
@@ -335,7 +335,6 @@ function ItemContent({ children }: TreeItemContentProps) {
               {showVisibility && (
                 <Button
                   variant='icon'
```

Comment:
```suggestion
          variant='icon'
          color='mono-bold'
```
Instead of removing the color variant, it now matches the 'mono-bold' button color.

You will also need to change the `design-toolkit/components/src/components/tree/styles.ts` file. Not ideal to have these coupled in this way, but...`color-info` needs to be `color-mono-bold` to match the color on the button for the selectors to work.

```
    visibility: [
      'group-not-visible/tree-item:enabled:color-mono-bold:fg-primary-muted',
      'group-not-viewable/tree-item:enabled:color-mono-bold:fg-primary-muted fg-primary-bold',
      'fg-primary-bold',
    ],
    expansion: [
      'group-not-visible/tree-item:enabled:color-mono-bold:fg-primary-muted',
      'group-not-viewable/tree-item:enabled:color-mono-bold:fg-primary-muted fg-primary-bold',
      'fg-primary-bold',
    ],
    drag: [
      'group-not-visible/tree-item:enabled:color-mono-bold:fg-primary-muted',
      'group-not-viewable/tree-item:enabled:color-mono-bold:fg-primary-muted fg-primary-bold',
      'fg-primary-bold',
    ],
```